tests/server-tests/api_test.go

```diff
@@ -47,17 +48,17 @@ func testMKCOL(t *testing.T, client *http.Client, dirname string) (string, error
 	req, err := newrequest(dirname, "MKCOL", nil)
 	if err != nil {
 		// handle error
-		t.Errorf("testMKCOL: Error: %v", t)
+		t.Errorf("testMKCOL: Error: %v", err)
```

Comment:
't' instead of 'err' was probably just a brain fart on my part, but what would 't' print here? Anything extra of value?